Deployment_files/kube-deployment_github

```diff
@@ -0,0 +1,37 @@
+apiVersion: v1
+kind: Pod
```

Comment:
delete this

Reply:
Only for testing, not deployed otherwise